src/locales/nl-SR.ts

```diff
@@ -0,0 +1,131 @@
+      { number: 67, value: 'ZEVENENZESTIG' },
+      { number: 66, value: 'ZESENZESTIG' },
+      { number: 65, value: 'VIJFENZESTIG' },
+      { number: 64, value: 'VIERENZESTIG' },
+      { number: 63, value: 'DRIEËNZESTIG' },
+      { number: 62, value: 'TWEEËNZESTIG' },
+      { number: 61, value: 'EENENZESTIG' },
+      { number: 60, value: 'ZESTIG' },
+      { number: 59, value: 'NEGENENVIJFTIG' },
+      { number: 58, value: 'ACHTENVIJFTIG' },
+      { number: 57, value: 'ZEVENENVIJFTIG' },
+      { number: 56, value: 'ZESENVIJFTIG' },
+      { number: 55, value: 'VIJFENVIJFTIG' },
+      { number: 54, value: 'VIERENVIJFTIG' },
+      { number: 53, value: 'DRIEËNVIJFTIG' },
+      { number: 52, value: 'TWEEËNVIJFTIG' },
+      { number: 51, value: 'EENENVIJFTIG' },
+      { number: 50, value: 'VIJFTIG' },
+      { number: 49, value: 'NEGENENVEERTIG' },
+      { number: 48, value: 'ACHTENVEERTIG' },
+      { number: 47, value: 'ZEVENENVEERTIG' },
+      { number: 46, value: 'ZESENVEERTIG' },
+      { number: 45, value: 'VIJFENVEERTIG' },
+      { number: 44, value: 'VIERENVEERTIG' },
+      { number: 43, value: 'DRIEËNVEERTIG' },
+      { number: 42, value: 'TWEEËNVEERTIG' },
+      { number: 41, value: 'EENENVEERTIG' },
+      { number: 40, value: 'VEERTIG' },
+      { number: 39, value: 'NEGENENDERTIG' },
+      { number: 38, value: 'ACHTENDERTIG' },
+      { number: 37, value: 'ZEVENENDERTIG' },
+      { number: 36, value: 'ZESENDERTIG' },
+      { number: 35, value: 'VIJFENDERTIG' },
+      { number: 34, value: 'VIERENDERTIG' },
+      { number: 33, value: 'DRIEËNDERTIG' },
+      { number: 32, value: 'TWEEËNDERTIG' },
+      { number: 31, value: 'EENENDERTIG' },
+      { number: 30, value: 'DERTIG' },
+      { number: 29, value: 'NEGENENTWINTIG' },
+      { number: 28, value: 'ACHTENTWINTIG' },
+      { number: 27, value: 'ZEVENENTWINTIG' },
+      { number: 26, value: 'ZESENTWINTIG' },
+      { number: 25, value: 'VIJFENTWINTIG' },
+      { number: 24, value: 'VIERENTWINTIG' },
+      { number: 23, value: 'DRIEËNTWINTIG' },
+      { number: 22, value: 'TWEEËNTWINTIG' },
+      { number: 21, value: 'EENENTWINTIG' },
+      { number: 20, value: 'TWINTIG' },
+      { number: 19, value: 'NEGENTIEN' },
+      { number: 18, value: 'ACHTTIEN' },
+      { number: 17, value: 'ZEVENTIEN' },
+      { number: 16, value: 'ZESTIEN' },
+      { number: 15, value: 'VIJFTIEN' },
+      { number: 14, value: 'VEERTIEN' },
+      { number: 13, value: 'DERTIEN' },
+      { number: 12, value: 'TWAALF' },
+      { number: 11, value: 'ELF' },
+      { number: 10, value: 'TIEN' },
+      { number: 9, value: 'NEGEN' },
+      { number: 8, value: 'ACHT' },
+      { number: 7, value: 'ZEVEN' },
+      { number: 6, value: 'ZES' },
+      { number: 5, value: 'VIJF' },
+      { number: 4, value: 'VIER' },
+      { number: 3, value: 'DRIE' },
+      { number: 2, value: 'TWEE' },
+      { number: 1, value: 'EEN' },
+      { number: 0, value: 'NUL' },
```

Comment:
Are all the words written in all capitals in this locale?

Reply:
Yes that is a deliberate choice, because for legal documents it's required to be all capital. If required I can change it to what is used in other locales and leave the capitalization to be implemented within business logic code.

Reply:
I believe all capital should be left for business logic to implement. You can make the changes as per other locales.